.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,51 +1,41 @@
 
 <!--
-^ Please summarise the changes you have done and explain why they are necessary here ^
-
-For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
-For new packages please briefly describe the package or provide a link to its homepage.
```

Comment:
I'm on the fence about this removal.

Yes, it makes the template shorter. However reminding/encouraging authors to link to changelogs/homepages/etc is valuable IMO.

Perhaps it'd be better to shorten and rephrase this instead of removing it?